doc/md/fundamentals/1-basic-syntax/5-characters-strings.md

```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 5
+---
+
+# Characters & strings
+
+## Characters
+
+The `Char` type in Motoko represents a single unicode character delimited with a single quotation mark (`'`).
+
+```motoko no-repl
+let letter : Char = 'A';
+let symbol : Char = '✮';
+
+// Comparing characters
+'I' == 'i' // False
+```
+
+## Strings
+
+Sequences of characters are handled using the [`Text`](https://internetcomputer.org/docs/motoko/base/Text) type, which represents immutable strings of unicode characters delimited with a double quotation mark (`"`).
+
+```motoko no-repl
+let greeting : Text = "Hello, world!";
+
+// Concatenating text
+
+"ICP" # "x" # "Motoko" // ICP x Motoko
+```
+
```

Comment:
````suggestion

`t.size()` returns the number of characters in the text `t`.

```motoko
"abc".size() == 3
```

`t.chars()` returns an iterator enumerating the characters in `t`. For example:

```motoko
import Char "mo:base/Char";

for (c in "abc".chars()) {
  Debug.print(Char.toText(c));
}
```

Text values can be compared using "==",  "<" and all the other relational operators.
````